docs/customer-portal/cp-did.md

```diff
@@ -5,8 +5,27 @@ A **DID** number is one that exists on the public telephone network. When dialed
 To edit **DID (Direct Inward Dial)**:
 
 1. Select one DID by clicking on the number, or select multiple and then click **`Bulk Edit`**.
+2. For single DID, click on number, click on **Edit** (beside Destination) and you will see the following fields in a new window.

```

Comment:
No need for "For single DID, click on number, "
As its already explained in step1